webpack.config.js

```diff
@@ -49,6 +51,12 @@ module.exports = async (env, options) => {
           from: "./src/functions/functions.json"
         }
       ]),
+      new CopyWebpackPlugin([
+        {
+          to: "taskpane.css",
```

Comment:
Is this missing from the underlying template? This isn't specific to adding test code.

Reply:
Oh, it is moved from another place. It would be better to limit this pull request to the test code changes. If the webpack config needs to be cleaned up, a separate change can be made.

Reply:
Yeah I think this was picked up in a merge